src/vos/vos_container.c

```diff
@@ -618,6 +649,30 @@ vos_cont_tab_register()
 	return rc;
 }
 
+int
+vos_dtx_update_resync_gen(daos_handle_t coh)
+{
+	struct vos_container	*cont;
+	struct vos_cont_df	*cont_df;
+	int			 rc;
+
+	cont = vos_hdl2cont(coh);
+	D_ASSERT(cont != NULL);
+
+	cont_df = cont->vc_cont_df;
+	rc = vos_tx_begin(vos_cont2umm(cont));
+	if (rc == 0) {
+		umem_tx_add_ptr(vos_cont2umm(cont),
```

Comment:
This can fail

Reply:
yes, but ignore umem_tx_add_ptr() return value is common case in current VOS implementation. Sometimes it is in cleanup logic, the caller may does not care about the return value. I would suggest to make another separated patch for related error handling.